### PR TITLE
Updated release notes

### PR DESCRIPTION
**Pull request by raghavsethi**

### CODE REVIEW

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 1]
```diff
@@ -8,3 +8,16 @@ General Changes
 * The deprecated casts between JSON and VARCHAR will now fail and provide the
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
+
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: No newline between items

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 2]
```diff
@@ -8,3 +8,16 @@ General Changes
...
+* Fix TypeUtils.hashPosition for structural types. Grouping by structural types
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: This should be explained in terms of impact to the end user. We don't put implementation details in release notes

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 3]
```diff
@@ -8,3 +8,16 @@ General Changes
...
+  will now work as expected.
+
+* The web interface now displays a lock icon next to users that have a principal
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: No need to wrap here

End sentence in period

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: How about "next to authenticated users."

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 4]
```diff
@@ -8,3 +8,16 @@ General Changes
...
+  set
+
+* The ``min_by`` and ``max_by`` aggregations now take a third parameter ``n``,
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: Link to the functions

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: No need for comma

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 5]
```diff
@@ -8,3 +8,16 @@ General Changes
...
   :doc:`/release/release-0.116`.
+
+* Fix TypeUtils.hashPosition for structural types. Grouping by structural types
+  will now work as expected.
+
+* The web interface now displays a lock icon next to users that have a principal
+  set
+
+* The ``min_by`` and ``max_by`` aggregations now take a third parameter ``n``, 
+  that will allow users to retrieve the values associated with the n smallest or
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: "that will allow users" is the wrong phrasing

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: "The ... aggregations now have an additional form that return multiple values."

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 6]
```diff
@@ -8,3 +8,16 @@ General Changes
...
+  largest keys.
+
+* Fixed 'IN' functionality for large sets (>1000 elements) of overloaded raw
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: This change was not merged

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 7]
```diff
@@ -8,3 +8,10 @@ General Changes
 * The deprecated casts between JSON and VARCHAR will now fail and provide the
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
+* Fixed bug in ``GROUP BY`` for structural types
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: This needs more explanation. Did it not work for certain things? Incorrect behavior? Bad performance?

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 8]
```diff
@@ -8,3 +8,10 @@ General Changes
...
+* The web interface now displays a lock icon next to authenticated users
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: End in sentence

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 9]
```diff
@@ -9,3 +9,10 @@ General Changes
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
 * * Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fixed ``GROUP BY`` to suport columns containing maps of structural types.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: Use "Fix" to be consistent with the others

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 10]
```diff
@@ -9,3 +9,10 @@ General Changes
...
+* The web interface now displays a lock icon next to authenticated users.
+* The ``min_by`` and ``max_by`` aggregations now take a third parameter ``n``, 
+  that will allow users to retrieve the values associated with the n smallest or
+  largest keys.
+* Fixed ``IN``functionality for large sets (>1000 elements) of overloaded raw
```
Comment (erichwang) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: space before functionality

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]:
```
* Fix incorrect results when using ``IN`` lists of more than 1000 elements on timestamp types.

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 11]
```diff
@@ -9,3 +9,10 @@ General Changes
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
 * * Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fixed ``GROUP BY`` to suport columns containing maps of structural types.
+* The web interface now displays a lock icon next to authenticated users.
+* The ``min_by`` and ``max_by`` aggregations now take a third parameter ``n``,
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: "The ... aggregations now have an additional form that return multiple values."

Use function link syntax for the functions

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 12]
```diff
@@ -9,3 +9,10 @@ General Changes
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
 * * Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fixed ``GROUP BY`` to suport columns containing maps of structural types.
+* The web interface now displays a lock icon next to authenticated users.
+* The ``min_by`` and ``max_by`` aggregations now take a third parameter ``n``, 
+  that will allow users to retrieve the values associated with the n smallest or
+  largest keys.
+* Fix ``IN`` functionality for large sets (>1000 elements) of overloaded raw
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]:
```
* Fix incorrect results when using ``IN`` lists of more than 1000 elements on timestamp types

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 13]
```diff
@@ -8,4 +8,8 @@ General Changes
 * The deprecated casts between JSON and VARCHAR will now fail and provide the
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
-* * Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fixed ``GROUP BY`` to suport columns containing maps of structural types.
+* The web interface now displays a lock icon next to authenticated users.
+* The :func:`min_by` and :func:`max_by` aggregations now have an additional form that return multiple values
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: These lines are a bit long, wrap to ~80 characters and end in period

File: presto-docs/src/main/sphinx/release/release-0.122.rst [hunk 14]
```diff
@@ -8,4 +8,8 @@ General Changes
 * The deprecated casts between JSON and VARCHAR will now fail and provide the
   user with instructions to migrate their query. For more details, see
   :doc:`/release/release-0.116`.
-* * Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fix ``NoSuchElementException`` when cross join is used inside ``IN`` query.
+* Fixed ``GROUP BY`` to suport columns containing maps of structural types.
```
Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: Use "Fix"

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]: "support"

Comment (electrum) [inline presto-docs/src/main/sphinx/release/release-0.122.rst]:
```
* Fix ``GROUP BY`` to support maps of structural types.

**Pull request merged**